tests/models/AuxiliaryChainRepository/create.test.ts

```diff
@@ -0,0 +1,111 @@
+// /<reference path="util.ts"/>
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import 'mocha';
+
+import {
+  AuxiliaryChainAttributes,
+  AuxiliaryChain,
+} from '../../../src/models/AuxiliaryChainRepository';
+
+import Database from '../../../src/models/Database';
+
+import Util from './util';
+
+import chai = require('chai');
+import chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const { assert } = chai;
+
+interface TestConfigInterface {
+  db: Database;
+}
+let config: TestConfigInterface;
+
+describe('AuxiliaryChainRepository::create', (): void => {
+  beforeEach(async (): Promise<void> => {
+    config = {
+      db: await Database.create(),
+    };
+  });
+
+  it('Checks creation of auxiliary chain model.', async (): Promise<void> => {
+    const auxiliaryChainAttributes: AuxiliaryChainAttributes = {
+      chainId: 10001,
+      originChainName: '10001',
+      ostGatewayAddress: '0x497A49648885f7aaC3d761817F191ee1AFAF399C',
+      ostCoGatewayAddress: '0x497A49648885f7aaB3d761817F191ee1AFAF399C',
+      anchorAddress: '0x3F45616cFb992988943ff3bA00e8c0aA46B4540a',
+      coAnchorAddress: '0xD1008015aA0Cf2a61493Bd19dE6C9ca88E934FCe',
```

Comment:
This address looks like actual addresses, for testing, we should use it like
`0x000000000000000000000000000000000000001`